README.md

```diff
@@ -150,6 +152,16 @@ const env = cleanEnv(process.env, {
 });
 ```
 
+### TypeScript users
+
+You can use either one of `makeBaseValidator`, `makeExactValidator` and `makeMarkupValidator`
+depending on your use case:
+
+- `makeBaseValidator<BaseT>` when you want the output to be narrowed-down to a subtype of `BaseT` (e.g. `str`).
+- `makeExactValidator<T>` when you want the output to be widened to `T` (e.g. `bool`).
+- `makeMarkupValidator` for input which can produce arbitrary output types (e.g. `json`).
+
+Note that `makeValidator` is an alias for `makeBaseValidator` which should cover most of use-cases.
```

Comment:
I don't think the alias is very helpful– and given backwards compat I think it should just be named `makeValidator`

Reply:
bfa3d416b8e4586a210cdc63034708d27a4f56ca